README.md

```diff
@@ -82,7 +82,7 @@ func main() {
 	if err != nil {
 		log.Fatalf("failed to create client, %v", err)
 	}
-	log.Fatal(c.StartReceiver(context.Background(), receive));
+	log.Println(c.StartReceiver(context.Background(), receive));
```

Comment:
nit: this would always exit 0 even in case of `error`. For correctness, I suggest handling the `error`:

```suggestion
	if err := c.StartReceiver(context.Background(), receive)); err != nil {...};
```